packages/config/src/environment.ts

```diff
@@ -27,5 +27,6 @@ export const env = cleanEnv(process.env, {
   COUNTRY_CONFIG_URL: url({ devDefault: 'http://localhost:3040/' }),
   MONGO_URL: url({ devDefault: 'mongodb://localhost/application-config' }),
   USER_MANAGEMENT_URL: url({ devDefault: 'http://localhost:3030/' }),
-  FHIR_URL: url({ devDefault: 'http://localhost:3447/fhir' })
+  FHIR_URL: url({ devDefault: 'http://localhost:3447/fhir' }),
+  HEARTH_MONGO_URL: str({ default: 'mongodb://localhost/hearth-dev' })
```

Comment:
@tumbledwyer if we cannot add this environment variable for HEARTH_MONGO_URL into docker-compose files in core in this PR, then we will need a companion PR in opencrvs-farajaland and opencrvs-countryconfig in order to add  into the docker compose files there

Reply:
That's correct. I think you need to add this both to docker-compose.yml in core and also in country-config and farajaland and also https://github.com/opencrvs/e2e

Reply:
Thanks for the catch. I've updated and created PRs for country config and faraland. 
e2e isn't letting me push a branch though...